PROTOCOL.md

```diff
@@ -301,31 +302,30 @@ Field    | Type | Description
 
 #### UnicastMessage
 
-A message addressed in response to a specific request. It contains a `StreamMessage` as a payload at the Message Layer level. The `StreamMessage` representation is also an array (nested in the `UnicastMessage` array) which is described in the [StreamMessage](#streammessage) section.
+A message addressed in response to a specific resend request. It contains a `StreamMessage` as a payload at the Message Layer level. The `StreamMessage` representation is also an array (nested in the `UnicastMessage` array) which is described in the [StreamMessage](#streammessage) section.
 
 ```
 [version, type, requestId, streamMessage]
 ```
 Example:
 ```
-[1, 1, "requestId", [...streamMessageFields]]
+[2, 1, "request-id", [...streamMessageFields]]
 ```
 
 Field    | Type | Description
 -------- | ---- | --------
-`requestId` | `string` | The request id to deliver the message to. Corresponds to the request id sent in a `ResendLastRequest`, a `ResendRangeRequest` or a `ResendFromRequest`.
```

Comment:
There's a few more instances of requestId listed in the fields, for various requests e.g.
```
requestId | string | ID of the request. Randomly generated by the sender.
```
Seems to be on some but not others, is this right?

Reply:
I guess `sender` here is a bit vague. If I understand correctly `requestId` should be equal to that of the original resend request (to which this is a response to).

Reply:
The `requestId` is defined collectively for all control layer messages in the beginning of the section. My purpose was to remove these message-specific rows (as some requests had a `requestId` before). I'll try to find all places where I missed them.